packages/flutter/lib/src/material/text_form_field.dart

```diff
@@ -105,6 +105,7 @@ class TextFormField extends FormField<String> {
     this.controller,
     String? initialValue,
     FocusNode? focusNode,
+    super.forceErrorText,
```

Comment:
Similarly then, what happens if you pass both forceErrorText and InputDecorator.errorText?

Reply:
If both forceErrorText and InputDecorator.errorText are passed, the value of forceErrorText will override InputDecorator.errorText. This behavior remains unchanged and is handled at line 212.

Reply:
Just to talk through an alternative, what if instead you pass InputDecorator.errorText to FormField.errorText?

Reply:
It's not practical to pass the errorText from InputDecorator to FormField's forceErrorText because FormField manages the logic to determine whether the form widget's state is valid or invalid, and it doesn't concern itself with the UI details. Since InputDecorator is responsible for displaying the form state in the UI, it is necessary to directly pass the errorText to InputDecorator to show errors. On the other hand, if we were to pass InputDecorator.errorText to FormField.forceErrorText, it would blur the lines between UI-specific implementation and form validation logic. InputDecorator should focus solely on rendering the decoration, and it shouldn't be tasked with forcing the Form into invalid state.